framework/src/media/audio/audio_manager.c

```diff
@@ -194,10 +192,11 @@ static audio_manager_result_t find_audio_card(audio_card_type_t card_type)
 	char type;
 	struct dirent *dir_entry;
 	DIR *dir_info;
-	audio_card_info_t *card = g_audio_out_cards;
+	audio_card_info_t dummy_card;
```

Comment:
I don't understand what below code means exactly, and I think this is not neccessary.

Reply:
Updated.